src/fn/soic.ts

```diff
@@ -33,6 +33,8 @@ export const extendSoicDef = (newDefaults: {
       } else if (!v.pl) {
         v.pl = v.pw! * (1.0 / 0.6)
       }
+      // Consider pin width when calculating pitch
+      v.p += v.pw
```

Comment:
nice

Reply:
hmm looking into this a bit, I think that we don't want to modify the pitch given by the user. We want to modify or fix the drawing function (if it's incorrect)

![image](https://github.com/user-attachments/assets/42782bb3-18fe-4bce-b188-feb8e29cce69)